endless/src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -5131,6 +5134,31 @@ bool CEndlessUsbToolDlg::CopyFilesToESP(const CString &fromFolder, const CString
 	return retResult;
 }
 
+bool CEndlessUsbToolDlg::CreatePersistentStorageFileOnexFAT(const CString& drive)
+{
+    HANDLE tmpfile;
+    CString storage = drive + L"\\endless\\persistent.img";
+    char marker[] = "endless_live_storage_marker";
+    uint64_t bytes;
+    ULARGE_INTEGER free_bytes;
+    DWORD written;
+
+    if (!GetDiskFreeSpaceEx(drive, &free_bytes, NULL, NULL))
+        return FALSE;
+
+    bytes = free_bytes.QuadPart;
+    tmpfile = CreateFile(storage, GENERIC_WRITE, FILE_SHARE_WRITE, NULL, CREATE_NEW, FILE_ATTRIBUTE_NORMAL, NULL);
+    if (tmpfile == INVALID_HANDLE_VALUE)
+        return FALSE;
+
+    // Ignore error because we're going to keep going anyway.
```

Comment:
Why? If we can't write the marker, the file is unusable as persistent storage. I think we should at least report the error and return `false` (after closing the handle), and maybe even try to delete the file?

Reply:
fair enough, just being lazy.  will fix.